### PR TITLE
Issue 41: Build our own Heapster image.

### PR DESCRIPTION
**Pull request by mwringe**

### CODE REVIEW

File: heapster-base/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+MAINTAINER Hawkular Metrics <hawkular-dev@lists.jboss.org>
+
+EXPOSE 8082
+
+ENV HEAPSTER_COMMIT=af4752e
+
+RUN yum install -y go git wget make && \
+    yum clean all
+
+RUN export GOPATH=/tmp/gopath && \
```
Comment (jcantrill) [inline heapster-base/Dockerfile]: Do we want to link this into the previous run statement?

Comment (mwringe) [inline heapster-base/Dockerfile]: We can. I usually like to keep things a bit separate if I think it makes sense logically (eg 1) setup system; 2) build heapster). I usually put a higher value of readability and separation of concerns versus having an extra layer.

Comment (jimmidyson) [inline heapster-base/Dockerfile]: Although after build you could remove the installed RPMs (go git wget make) & hence make a smaller image.

**Comment by mwringe** [comment]
Ok, its been updated to included the requested changes.

**Comment by jcantrill** [comment]
LGTM

**Pull request merged**